app/handlers/karma.py

```diff
@@ -45,7 +51,23 @@ async def get_top(
     user: User,
     chat_repo: ChatRepo,
     user_repo: UserRepo,
+    config: Config,
 ):
+    parts = message.text.split(maxsplit=1)
+    if len(parts) > 1:
+        bot_reply = await message.reply(
+            "Просмотр топа другого чата возможен только в личных сообщениях с ботом"
+        )
+
+        asyncio.create_task(
+            cleanup_command_dialog(
+                bot_message=bot_reply,
+                delete_bot_reply=True,
+                delay=config.time_to_remove_temp_messages,
```

Comment:
Do we have that key in config?

Reply:
https://github.com/bomzheg/KarmaBot/blob/master/app/models/config/__init__.py#L27